x/emissions/proto/emissions/v9/events.proto

```diff
@@ -436,3 +434,146 @@ message EventPruneRecords {
   int64 block_height = 1;
   uint64 topic_id = 2;
 }
+
+// Stake removal processing events
+message EventReputerStakeRemovalCompleted {
+  uint64 topic_id = 1;
+  string reputer = 2;
+  string amount = 3 [
+    (cosmos_proto.scalar) = "cosmos.Int",
+    (gogoproto.customtype) = "cosmossdk.io/math.Int",
+    (gogoproto.nullable) = false,
+    (amino.dont_omitempty) = true
+  ];
+  int64 block_height = 4;
+}
+
+message EventDelegateStakeRemovalCompleted {
+  uint64 topic_id = 1;
+  string reputer = 2;
+  string delegator = 3;
+  string amount = 4 [
+    (cosmos_proto.scalar) = "cosmos.Int",
+    (gogoproto.customtype) = "cosmossdk.io/math.Int",
+    (gogoproto.nullable) = false,
+    (amino.dont_omitempty) = true
+  ];
+  int64 block_height = 5;
+}
+
+message EventDelegateRewardShareUpdated {
```

Comment:
Is it useful to track that? Being curious because I'd consider this as an internal thing only